Pod/Classes/Runtime.swift

```diff
@@ -42,28 +28,94 @@ public class Runtime{
     @param host,user,password,organization.
     @return Promise<Bool>.
     */
-    
     public func start(host:String , user: String , password:String, organization:String) -> Promise<Bool> {
+        return start(["host": host, "username": user, "password": password, "organization": organization])
+    }
+    
+    
+    /**
+    Initialize the SDK and connect to EXP.
+    @param options
+    @return Promise<Bool>.
+    */
+    public func start(options:[String:String]) -> Promise<Bool> {
         
         return Promise { fulfill, reject in
-            hostUrl=host
-            login(user, password, organization).then {(token: Token) -> Void  in
-                tokenSDK = token.token
-               
+            if let host = options["host"] {
+                hostUrl=host
+            }
+            
+            if let user = options["username"], password = options["password"], organization = options["organization"] {
+                login(user, password, organization).then {(token: Token) -> Void  in
+                    tokenSDK = token.token
+                    
+                    // Creating an Instance of the Alamofire Manager
+                    var manager = Manager.sharedInstance
+                    // Specifying the Headers we need
+                    manager.session.configuration.HTTPAdditionalHeaders = [
+                        "Authorization": "Bearer " + tokenSDK
+                    ]
+                    
+                    
+                    socketManager.start_socket().then { (result: Bool) -> Void  in
+                        if result{
+                            fulfill(true)
+                        }
+                    }
+                }
+            }
+            
+            if let deviceUuid = options["uuid"], secret = options["secret"] {
```

Comment:
change the variable name to uuid bc in line 85 has the same name for the variable